packages/liveblocks-react/src/comments/lib/selected-threads.ts

```diff
@@ -30,6 +30,11 @@ export function selectedThreads<M extends BaseMetadata>(
       const query = options.query;
       if (!query) return true;
 
+      // If the query includes 'resolved' filter and the thread's 'resolved' value does not match the query's 'resolved' value, exclude the thread
+      if (query.resolved !== undefined && thread.resolved !== query.resolved) {
+        return false;
+      }
```

Comment:
@nvie I noticed we do not check for the resolved property inside `query` when filtering for threads. They may have been missed during the change to make `resolved` as first class citizen.